### PR TITLE
some logic for the category + quizzes

### PR DESCRIPTION
**Pull request by ManarAbdelkarim**

### CODE REVIEW

File: Quiz.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (NizarAlsaeed) [inline Quiz.json:1]: the team never used json files, we barely know them, so either avoid them or give a simple illustration to us why and when should we use them

File: category.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<body id="change-background">
+    <header></header>
+    <main>
+<table>
+
+</table>
+
+    </main>
+    <footer></footer>
+    <!-- <script type="text/javascript" src="./js/Quiz.json"></script> -->
```
Comment (NizarAlsaeed) [inline category.html:19]: do not push commented lines

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/Style.css">
+    <title>QUIZY BUZZY</title>
+</head>
+<body>
+   <header>
+       <section class="nav"><nav id="nav"></nav></section>
+       <section class="second-header"><div id="second-header"></div></section>
```
Comment (NizarAlsaeed) [inline index.html:13]: we should not use the same name twice for a class and an id,

File: js/App.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+let table = document.getElementsByTagName('table')[0];
+
+table.addEventListener('click',changeColor);
+function changeColor(event){
+    debugger;
```
Comment (NizarAlsaeed) [inline js/App.js:5]: do not push debugger :)

**Review by NizarAlsaeed — COMMENTED** [review]
Great work Manar,
i think we need more comments on the code, we have a long hard job to understand uncommented loops, conditions and functions
help us please ;(

File: Quiz.json (hunk 1 above)
Comment (ManarAbdelkarim) [reply Quiz.json:1]: alright

File: index.html (hunk 3 above)
Comment (ManarAbdelkarim) [reply index.html:13]: got it ..  will be changed later

File: js/App.js (hunk 4 above)
Comment (ManarAbdelkarim) [reply js/App.js:5]: sorry I thought I deleted all debuggers (:

File: category.html (hunk 2 above)
Comment (ManarAbdelkarim) [reply category.html:19]: I also thought I deleted all comments ( X

**Pull request merged**